viz_scripts/bin/generate_plots.py

```diff
@@ -77,8 +86,7 @@ def compute_for_date(month, year):
         study_type=dynamic_config['intro']['program_or_study'],
         mode_of_interest=mode_studied,
         include_test_users=dynamic_config.get('metrics', {}).get('include_test_users', False),
-        has_dynamic_labels = has_dynamic_labels,
-        dynamic_labels_url = dynamic_labels_url)
+        dynamic_labels = dynamic_labels)
```

Comment:
it seems to me that you should have a default value for `dynamic_labels`. Please indicate "Testing done" **for `generate_plots.py`**

Reply:
Yes, I have initialized the dynamic_labels as `dynamic_labels = { }` for default value.

Reply:
you have initialized the dynamic labels inside the notebook. As far as I can tell, you haven't initialized the dynamic labels in `generate_plots.py`.

quick proof of what happens when you try to use a variable that is not defined.

```
$ python3
>>> print(dynamic_labels)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'dynamic_labels' is not defined
```

Again,

> Please indicate "Testing done" **for `generate_plots.py`**

Reply:
Initialized the dynamic_labels to { } beforehand. Since, not all json would have dynamic configs, so it felt appropriate to initialize with { }. Added reference to https://github.com/e-mission/nrel-openpath-deploy-configs/blob/main/label_options/example-study-label-options.json to initialize it properly, when required.